lib/lrmd/lrmd_client.c

```diff
@@ -1315,9 +1321,13 @@ lrmd__init_remote_key(gnutls_datum_t *key)
     }
 
     // Try fallback location, if environment wasn't set to it and default failed
+    // @COMPAT Drop at 3.0.0
     if (env_is_fallback) {
         alt_rc = env_rc;
     } else if (default_rc != pcmk_rc_ok) {
+        crm_warn("Attempting to read Pacemaker Remote key from %s. "
+                 "This undocumented behavior is deprecated and will be "
+                 "removed in a future release.", ALT_REMOTE_KEY_LOCATION);
```

Comment:
I don't want to warn when we try, only if we find something (if they don't have a key at one of the deprecated locations, they're not doing anything wrong)